tests/Common/Common.h

```diff
@@ -7,24 +7,58 @@
 #include <string>
 #include <vector>
 
-#pragma pack(4)
-class TestPacking
+#pragma pack(1)
+class DLL_API TestPacking1
 {
 public:
-    int integer;
-    bool boolean;
+    bool b1;
+    bool b2;
     double dble;
-    int i2;
+
+    TestPacking1();
+    TestPacking1(int a, int b, double d);
+    ~TestPacking1();
 };
+#pragma pack()
 
 #pragma pack(2)
-struct TestP 
+class DLL_API TestPacking2
 {
-    char b;
-    bool c;
-    double d;
+public:
+    double a;
```

Comment:
Make all fields in the 4 classes the same. The point is to have different sizes and offsets for the same fields so that we know the difference only comes from packing.